### DISCUSSION TITLE
Custom installation directory

### DISCUSSION THREAD

**Discussion by matkoch** [Homebrew/discussions#3464] [Getting started]
I'd like to `brew install` into a specific location without making the tool (for instance `imagemagick`) available globally. This is to allow several scripts to depend on individual versions of the tool. I read somewhere, that I could set `HOMEBREW_PREFIX`, but that didn't change anything in `brew config`.

**Comment by SMillerDev** [comment]
You'll needed to reinstall brew into that directory. Which will probably require you to build everything from source.
